ci/run-tests.ps1

```diff
@@ -24,14 +24,26 @@ function Force-ResolvePath {
 if ($run_with_spatial) {
     # Generate schema and snapshots
     Write-Output "Generating snapshot and schema for testing project"
+    Start-Process "$unreal_editor_path" -Wait -PassThru -NoNewWindow -ArgumentList @(`
+        "$uproject_path", `
+        "-SkipShaderCompile", # Skip shader compilation
+        "-nopause", # Close the unreal log window automatically on exit
+        "-nosplash", # No splash screen
+        "-unattended", # Disable anything requiring user feedback
+        "-nullRHI", # Hard to find documentation for, but seems to indicate that we want something akin to a headless (i.e. no UI / windowing) editor
+        "-run=CookAndGenerateSchema", # Run the commandlet
+        "-map=`"$test_repo_map`"" # Which maps to run the commandlet for
+        "-targetplatform=LinuxServer"
+    )
+    
     Start-Process "$unreal_editor_path" -Wait -PassThru -NoNewWindow -ArgumentList @(`
         "$uproject_path", `
         "-NoShaderCompile", # Prevent shader compilation
         "-nopause", # Close the unreal log window automatically on exit
         "-nosplash", # No splash screen
         "-unattended", # Disable anything requiring user feedback
         "-nullRHI", # Hard to find documentation for, but seems to indicate that we want something akin to a headless (i.e. no UI / windowing) editor
-        "-run=GenerateSchemaAndSnapshots", # Run the commandlet
+        "-run=GenerateSnapshot", # Run the commandlet
```

Comment:
The removal of `GenerateSchemaAndSnapshots` is a breaking change for our customers CI. Either we use the old name to prevent this and mark it as deprecated or we need a breaking change release note.

Reply:
I added back the deprecated commands, with appropriate logs.